oauth-keycloak-authorizer/src/main/java/io/strimzi/kafka/oauth/server/authorizer/AuthzConfig.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017-2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.oauth.server.authorizer;
+
+import io.strimzi.kafka.oauth.common.Config;
+
+import java.util.Properties;
+
+public class AuthzConfig extends Config {
+
+    public static final String STRIMZI_AUTHZ_CLIENT_ID = "strimzi.authz.client.id";
```

Comment:
I wonder if we should use `authorization` intead of `authz`. Lot of people does not know / use this as shortcut and might be confused.

Reply:
Fixed.